src/index.js

```diff
@@ -1,4 +1,9 @@
 import './style.css';
 import UI from './UI.js';
+import fetchData from './display-list-of-shows.js';
 
-UI.headerLogo();
\ No newline at end of file
+UI.headerLogo();
```

Comment:
[OPTIONAL]
I suggest that this would be better placed in the DOMContentLoaded below. I got this tip from a reviewer once. He told me that everything that is to be displayed is based placed in a `windows.onload` or a `DOMConentLoaded` such that the browser always loads the script first

Reply:
Okay, I'll do that. 👍